core/capabilities/ccip/ccipevm/gas_helpers.go

```diff
@@ -85,5 +94,6 @@ func (gp EstimateProvider) CalculateMessageMaxGas(msg cciptypes.Message) uint64
 		SupportsInterfaceCheck +
```

Comment:
do we need to also add the `destGasOverhead`?

https://github.com/smartcontractkit/chainlink/blob/23452266132228234312947660374fb393e96f1a/contracts/src/v0.8/ccip/FeeQuoter.sol#L99

Reply:
How important is this to have @matYang? I'll add a TODO and create a ticket in the meantime.